### PR TITLE
use FALLBACK method in client

### PR DESCRIPTION
**Pull request by grondilu**
so that the fall-back mechanism can be inherited

### CODE REVIEW

File: lib/JSON/RPC/Client.rakumod [hunk 1]
```diff
@@ -109,6 +84,7 @@ method !handler( Str :$method!, :$params ) {
     # to be used during the invocation of the method.
     # This member MAY be omitted.
     %request{'params'} = $_ with $params;
+    note %request.raku;
```
Comment (grondilu) [inline lib/JSON/RPC/Client.rakumod:87]: oops that line was for debugging.  should be removed.

**Comment by bbkr** [comment]
When original code was written there was no FALLBACK sub available, injecting it through meta was the only way to add it.

Your version is not only inheritable but also it is more clean approach.

Before I merge it I think you left some debug info in the code.

**Comment by bbkr** [comment]
You can simply push commit to patch-1 branch in grondilu/JSON-RPC.

Pull requests are always merging current state, not the one when PR was created.

**Comment by grondilu** [comment]
> You can simply push commit to patch-1 branch in grondilu/JSON-RPC.
> 
> Pull requests are always merging current state, not the one when PR was created.

Noted.  Done.

**Comment by bbkr** [comment]
Thanks. I'll make version release from your changes in ~3 hours after my $dayjob.

**Pull request merged**